Brownfield cloud guide (#858)
tinacms/tinacms.org: content/guides/tina-cloud/existing-site/create-app.md

---
title: Create App in Tina Cloud
last_edited: '2021-05-10T10:00:00.000Z'
---

In this step, we will be setting up a Tina Cloud account, and creating an "app".

In Tina Cloud, Apps connect Tina Cloud with a GitHub repository.
An app is the connection point between your site and your users, allowing authorized users to login, and start making edits to your site's content.

## Sign up for Tina Cloud

You can register for Tina Cloud by creating a free organizatiion [here](https://auth.tina.io/register).

## Create an app

> Once registered, you can log in and see the dashboard. We will want to create an app.

Reviewer: ```suggestion
Once registered, you can log in to your organization and access your dashboard. Let's create an app.
```
